### PR TITLE
[Docs] Fixed responsiveness 

### LINKED ISSUE #8029: [Docs]Responsiveness issues
I have been spending my time reading about meshery and have encountered a few issues  . The more I'll dive deep about project , the more issues shall be listed here😆

#### Current State

- [x]  [Responsive issue of announced vulnerabilities](#second_issue)
- [x]  Responsive issue in supported platforms
- [x] [Incomplete navigation components](#third-issue)
- [x] [The calender in the footer doesn't turns to blue on hover as expected](#last_issue)
- [x]  [Missing grafana boards svg]

## 

The current sidebar components headings have incomplete subheadings (for example in concepts) [I dont know its a bug or feature]
| Shown Subheadings | Complete subheadings |
| --- | ----------- |
| ||
##
#last_issue

##
The headings should have complete subheadings
##

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Ghat0tkach**
**Notes for Reviewers**

This PR fixes #8029 

Added responsiveness for both vulnerability and supported platforms table.
Added missing subcontents on the sidebar

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Ghat0tkach** [comment]
@vishalvivekm @thisiskaransgit  , please review the changes .

File: docs/_data/toc.yml [hunk 1]
```diff
@@ -67,18 +67,32 @@
...
+      url: tasks/filter-management
     - title: Lifecycle Management
       url: tasks/lifecycle-management
     - title: Pattern Management
       url: tasks/pattern-management
     - title: Performance Management
       url: tasks/performance-management
-
+    - title: Meshery and SMI Management
+      url: tasks/service-mesh-interface
```
Comment (leecalcote) [inline docs/_data/toc.yml:95]:
```suggestion
#    - title: Meshery and SMI Management
#      url: tasks/service-mesh-interface  
```

File: docs/_data/toc.yml [hunk 2]
```diff
@@ -67,18 +67,32 @@
       url: concepts/architecture
     - title: Adapters
       url: concepts/architecture/adapters
+    - title: Broker
```
Comment (leecalcote) [inline docs/_data/toc.yml:70]: "under Architecture" meaning indented under Architecture

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit f362d991360098ec66d2c776920dd93d6e59791a at: https://64aae56b51b999484372f396--meshery-docs-preview.netlify.app

File: docs/_data/toc.yml (hunk 2 above)
Comment (Ghat0tkach) [reply docs/_data/toc.yml]: apologies, Done!
added rest of them under architecture except catalog subheading since its location is concepts/catalog

**Comment by l5io** [comment]
🚀 Preview for commit c119197aa48bbca0953f1b8101fa96c8e13c7736 at: https://64aae96dc6d6e648f3dcbe02--meshery-docs-preview.netlify.app

**Comment by sudhanshutech** [comment]
@Ghat0tkach Let's discuss this on Websites Meeting call on Monday. Please add this as an agenda item in the [meeting mintes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would

File: docs/_data/toc.yml [hunk 3]
```diff
@@ -63,22 +63,38 @@
...
       url: concepts/architecture
+      children:
+        - title: Broker
+          url: concepts/architecture/broker
+        - title: Database
+          url: concepts/architecture/database  
+        - title: Meshsync
+          url: concepts/architecture/meshsync     
+        - title: Operator
+          url: concepts/architecture/operator
```
Comment (leecalcote) [inline docs/_data/toc.yml:76]:
```suggestion
          url: concepts/architecture/operator
        - title: Adapters
          url: concepts/architecture/adapters          
```

File: docs/_data/toc.yml [hunk 4]
```diff
@@ -63,22 +63,38 @@
 - title: 🧠 Concepts
   url: concepts
   links:
-    - title: Architecture
+    - title: Architecture ↆ
       url: concepts/architecture
+      children:
```
Comment (leecalcote) [inline docs/_data/toc.yml:68]:
```suggestion
      children:
        - title: Adapters
          url: concepts/architecture/adapters      
```

**Review by leecalcote — APPROVED** [review]
Good!

**Comment by Ghat0tkach** [comment]
Thank You

**Pull request merged**